tests/e2e/initialization/config.go

```diff
@@ -105,6 +109,24 @@ var (
 	WalletFeeTokens = sdk.NewCoin(E2EFeeToken, sdk.NewInt(WalletFeeBalance))
 )
 
+// START: CAN REMOVE POST v17 UPGRADE
+func strAllUpgradeBaseDenoms() string {
+	upgradeBaseDenoms := fmt.Sprintf("%s%s,", DaiBalanceA, DaiDenom)
+	n := len(AssetPairs)
+	for i, assetPair := range AssetPairs {
+		if assetPair.BaseAsset == "uion" {
```

Comment:
why do we pass for uion?

Reply:
It is already registered in another step. If I removed it from that step and added it here, it would have been more difficult to unwind this change (this change is only needed for v17 and can be removed after)